src/gui/properties/peerlistdelegate.h

```diff
@@ -67,57 +68,57 @@ class PeerListDelegate: public QItemDelegate {
 
     ~PeerListDelegate() {}
 
-    void paint(QPainter * painter, const QStyleOptionViewItem & option, const QModelIndex & index) const
+    void paint(QPainter *painter, const QStyleOptionViewItem &option, const QModelIndex &index) const override
     {
         painter->save();
+
         const bool hideValues = Preferences::instance()->getHideZeroValues();
         QStyleOptionViewItem opt = QItemDelegate::setOptions(index, option);
-        switch(index.column()) {
+        QItemDelegate::drawBackground(painter, opt, index);
+
+        switch (index.column()) {
         case PORT: {
-            QItemDelegate::drawBackground(painter, opt, index);
             opt.displayAlignment = Qt::AlignRight | Qt::AlignVCenter;
             QItemDelegate::drawDisplay(painter, opt, option.rect, index.data().toString());
-            }
-            break;
+        }
```

Comment:
damn, uncrustify didn't indent correctly.

>Coding guidelines need an update?

That would require a new issue ticket for discussion.

>If yes could you also move #include "base/bittorrent/infohash.h" one line up?

Not suitable for this PR.